src/utils/_contains-polyfill.js

```diff
@@ -0,0 +1,19 @@
+// Source: https://github.com/Financial-Times/polyfill-service/pull/183/files
```

Comment:
can we have these files in polyfills folder under utils ?

Reply:
Moved